scanner/image/db/konflux.Dockerfile

```
@@ -0,0 +1,50 @@
+FROM registry.redhat.io/rhel8/postgresql-15:latest
+
+LABEL \
+    com.redhat.component="rhacs-scanner-v4-db-container" \
+    com.redhat.license_terms="https://www.redhat.com/agreements" \
+    description="Scanner v4 Database Image for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.description="Scanner v4 Database Image for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.display-name="scanner-v4-db" \
+    io.openshift.tags="rhacs,scanner-db,stackrox" \
```

Cosmetic suggestion:
```suggestion
    io.openshift.tags="rhacs,scanner-v4-db,stackrox" \
```

---

f0ac2d4